consensus/spos/bls/subroundEndRound_test.go

```diff
@@ -62,6 +64,45 @@ func initSubroundEndRoundWithContainer(
 		appStatusHandler,
 		&testscommon.SentSignatureTrackerStub{},
 		&mock.SposWorkerMock{},
+		&dataRetrieverMocks.ThrottlerStub{},
+	)
+
+	return srEndRound
+}
+
+func initSubroundEndRoundWithContainer400Sig(
```

Comment:
maybe rename the method - I see no constraint over there being 400 sigs.
I could call this method for a dfferent number of signatures, and it would be misleading then.
Could be renamed to `initSubroundEndRoundWithContainerAndConsensusState`